platforms/forge/src/main/java/com/pg85/otg/forge/events/client/ClientFogHandler.java

```diff
@@ -0,0 +1,307 @@
+package com.pg85.otg.forge.events.client;
```

Comment:
Awesome! Where'd you get the info/inspiration/code from to write this? Any other mods or MC code you looked at? Might be nice to add a reference (if any).